lib/paper_trail/attribute_serializers/cast_attribute_serializer.rb

```diff
@@ -32,15 +34,15 @@ def defined_enums
       # This implementation uses AR 5's `serialize` and `deserialize`.
       class CastAttributeSerializer
         def serialize(attr, val)
-          @klass.type_for_attribute(attr).serialize(val)
+          PaperTrail::AttributeSerializers::AttributeSerializer.for(@klass, attr).serialize(val)
```

Comment:
It's not necessary to fully-qualify the constant here. `AttributeSerializer` alone is sufficient.